packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ProxyEOA.sol

```diff
@@ -113,9 +103,9 @@ contract OVM_ProxyEOA {
     )
         internal
     {
-        Lib_SafeExecutionManagerWrapper.safeSSTORE(
-            IMPLEMENTATION_KEY,
-            Lib_Bytes32Utils.fromAddress(_implementation)
-        );
+        bytes32 addr32 = Lib_Bytes32Utils.fromAddress(_implementation);
```

Comment:
I double checked the test files to make sure these are aligned correctly, looks good.